src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -20,15 +35,18 @@ function transformApplicants(applicants) {
   applicants.forEach(app => {
     const transformedApp = {
       fullName: app.applicantName ?? '',
-      sshOrTin: app.applicantSSN ?? '',
-      dateOfBirth: app.applicantDOB ?? '',
+      ssnOrTin: app.applicantSSN?.ssn ?? '',
+      dateOfBirth: fmtDate(app.applicantDOB) ?? '',
       phoneNumber: app.applicantPhone ?? '',
       email: app.applicantEmailAddress ?? '',
       vetRelationship: transformRelationship(
-        app.applicantRelationshipToSponsor?.relationshipToVeteran || 'NA',
+        app.applicantRelationshipToSponsor || 'NA',
       ),
       sponsorMarriageDetails:
         app?.applicantSponsorMarriageDetails?.relationshipToVeteran || 'NA',
+      isEnrolledInMedicare:
+        app?.applicantMedicareStatus?.eligibility === 'enrolled',
+      hasOtherHealthInsurance: app?.applicantHasOhi?.hasOhi === 'yes',
```

Comment:
These had been previously added to the front-end but not updated here in the submit transformer. Adding now.